admin/spec/controllers/spree/admin/store_credit_categories_spec.rb

```
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+RSpec.describe Spree::Admin::StoreCreditCategoriesController, type: :controller do
```

RuboCop (1.73)

[convention] Spec path should end with `spree/admin/store_credit_categories_controller*_spec.rb`.

(RSpec/SpecFilePathFormat)